topics/go/language/struct_types/example2/example2.go

```diff
@@ -34,7 +34,10 @@ func main() {
 
 	// Display the values.
 	fmt.Printf("%+v\n", e2)
-	fmt.Println("Flag", e2.flag)
-	fmt.Println("Counter", e2.counter)
-	fmt.Println("Pi", e2.pi)
+	fmt.Printf(
```

Comment:
Please revert this one. The point in this example is learning how to get/set struct field values using the `.` operator. I think students will be distracted by the printf verbs.